src/main/java/com/sigopt/model/Experiment.java

```diff
@@ -62,6 +62,10 @@ public Integer getObservationBudget() {
       return Utils.asInteger(this.get("observation_budget"));
     }
 
+    public boolean getDevelopment() {
```

Comment:
to be consistent with the other getters, this should return a `Boolean` (since it could plausibly be null) but the setter should enforce that a valid `boolean` is set